dotCMS/src/main/java/com/dotcms/graphql/datafetcher/FieldDataFetcher.java

```diff
@@ -48,10 +55,42 @@ public Object get(final DataFetchingEnvironment environment) throws Exception {
                 }
             }
 
-            return fieldValue;
+            final boolean renderField = Try.of(()-> (boolean) environment.getArgument("render"))
+                    .getOrElse(false);
+
+            return renderField && isFieldRenderable(field)
+                    ? renderFieldValue(environment, fieldValue)
+                    : fieldValue;
         } catch (Exception e) {
             Logger.error(this, e.getMessage(), e);
             throw e;
         }
     }
+
+    private Object renderFieldValue(DataFetchingEnvironment environment, Object fieldValue) {
+        final String fieldValueAsStr = fieldValue.toString();
+
+        final HttpServletRequest request = ((DotGraphQLContext) environment.getContext())
```

Comment:
I prefer the DotGraphQLContext.class.cast(environment.getContext()). getHttpServletRequest()

But it is pretty much the same